### ISSUE TITLE
VideoPress videos failing (Either present or after being uploaded) 

### ISSUE THREAD

**Issue by DavidSaladu** [Automattic/wp-calypso#77741]
### Quick summary

Still adding more info and figuring out patterns but so far:

### Steps to reproduce

1. On a site add a video through media
2. Let it be reprocessed
3. Check the video

### What you expected to happen

After it gets uploaded and reprocessed the video should appear normally

### What actually happened

The video showcases an error:

![image](https://github.com/Automattic/wp-calypso/assets/79575219/1c07aeb6-e3b8-4739-80a9-c767215152d7)

The file url appears as not found (404).
URL

### Impact

Some (< 50%)

### Available workarounds?

No and the platform is unusable

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

Macbook Pro Ventura 13.4
Chrome Version 113.0.5672.126 (Official Build) (x86_64)
Tested so far with Premium and Business plans with the same result.

**Comment by DavidSaladu** [comment]
6359257-zd-woothemes
6359456-zd-woothemes

**Comment by dev-sabbir** [comment]
6359358-zd-woothemes

**Comment by ezebecke** [comment]
Also reported on 6359555-zd-woothemes and 6359522-zd-woothemes

**Comment by kylemcph** [comment]
Another report here: 6359396-zd-woothemes

**Comment by john-legg** [comment]
📌 REPRODUCTION RESULTS

Tested on Simple – Not applicable (VideoPress is Premium plan and above)
Tested on Atomic – Could Not Replicate
Replicable outside of Dotcom – No

📌 FINDINGS/SCREENSHOTS/VIDEO

This issue was fixed with a revert according to this thread: p1685697412689189-slack-C02LT75D3

However, I still wanted to test this to make sure everything is working properly. I tested this on Atomic sites (WPCOM and self-hosted) and the video upload, reprocessing, and playback worked as expected. Since VideoPress only available on Premium plans and above, Simple sites were not tested.

<img width="1278" alt="VideoPress Test" src="https://github.com/Automattic/wp-calypso/assets/96805492/f0935fc6-1057-4822-807e-fe60fdfa4209">

Note: I found a separate issue while testing VideoPress on eCommerce that I will open shortly.

📌 ACTIONS

Closed as resolved by revert (p1685697412689189-slack-C02LT75D3)
